src/main/java/org/folio/rest/impl/NoteTypesImpl.java

```diff
@@ -66,17 +75,17 @@ public void getNoteTypes(String query, int offset, int limit, String lang, Map<S
     });
   }
 
+  @Validate
   @Override
   public void postNoteTypes(String lang, NoteType entity, Map<String, String> okapiHeaders,
-                            Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-
-    asyncResultHandler.handle(succeededFuture(PostNoteTypesResponse.status(Response.Status.NOT_IMPLEMENTED).build()));
+      Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    PgUtil.post(NOTE_TYPE_TABLE, entity, okapiHeaders, vertxContext, PostNoteTypesResponse.class, asyncResultHandler);
```

Comment:
Would we need to call removeNotStoredFields(entity) on POST (similar to what is in place for PUT)?

Reply:
this is not need because the field is read only and as a result annotated with @javax.validation.constraints.Null
which in turn resets it to null

I created a small test to verify the above:
'''
@Test
  public void test() {
    try {
      NoteType creating = nextRandomNoteType().withUsage(new NoteTypeUsage().withNoteTotal(10));
      postWithStatus("note-types/", toJson(creating), HttpStatus.SC_CREATED);
    } finally {
      DBTestUtil.deleteAllNoteTypes(vertx);
    }
  }
'''
In debug mode, inside `NoteTypesImpl.postNoteTypes()` I see that `usage` indeed equals to null:
![image](https://user-images.githubusercontent.com/42244720/55867629-8adf9100-5b8b-11e9-9bb6-923bf8798b67.png)

Reply:
see also this section from RMB wiki:
https://github.com/folio-org/raml-module-builder#json-schema-fields